libindic/payyans/maps/panchari.map

```diff
@@ -149,3 +149,5 @@ z=ന്മ
 Z=ള്‍
 Þ=ാ
 µ=ക
+,=ച്ച
+-=
```

Comment:
- is also mapped to -ഹ്ന on line 7

Reply:
Actually, the - on line 7 is a different character (hyphen minus and soft hyphen)
![image](https://user-images.githubusercontent.com/945777/81663340-3c6e8700-945c-11ea-820a-247f2ac4216d.png)
Sorry for the confusion. This should be merged.